instagram-app/src/components/Login/Login.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react'
+import './Login.scss'
+
+export default class Login extends Component {
+    state={
+        username:'',
+        pass:''
+    }
+
+    render() {
+        return (
+            <section className="login container">
+                <fieldset 
+                    className="d-flex flex-column mt-2"
+                >
+                    <legend>Login</legend>
+                    <div className="form-group row">
+                        <label htmlFor="username" className="col-form-label col-2">
+                            User: 
+                        </label>
+                        <div className="col-10">
+                            <input 
+                                type="text" 
+                                name="username" 
+                                value={this.state.username} 
+                                onChange={this.formEdit}
+                                className="form-control"
+                            />
+                        </div>
+                    </div>
+                    <div className="form-group row">
+                        <label htmlFor="password" className="col-form-label col-2">
+                            Password: 
+                        </label>
+                        <div className="col-10">
+                            <input 
+                                type="password" 
+                                name="pass" 
+                                value={this.state.pass} 

```

Comment:
Why not build these into a form with an on submit? Did you build in another one of those nifty submit handler conditions that tests if the key that was pressed was enter and submit if that test is true?